tests/src/main/java/net/neoforged/neoforge/debug/data/CustomFeatureFlagsTests.java

```diff
@@ -122,13 +123,13 @@ protected void buildRecipes() {
                         shapeless(RecipeCategory.MISC, Items.DIAMOND)
                                 .requires(ItemTags.DIRT)
                                 .unlockedBy("has_dirt", has(ItemTags.DIRT))
-                                .save(output.withConditions(FlagCondition.isEnabled(flag)), enabledRecipeName);
+                                .save(output.withConditions(FlagCondition.of(flag)), enabledRecipeName);
 
                         // recipe available when above flag is disabled
                         shapeless(RecipeCategory.MISC, Items.DIRT)
                                 .requires(Tags.Items.GEMS_DIAMOND)
                                 .unlockedBy("has_diamond", has(Tags.Items.GEMS_DIAMOND))
-                                .save(output.withConditions(FlagCondition.isDisabled(flag)), disabledRecipeName);
+                                .save(output.withConditions(new NotCondition(FlagCondition.of(flag))), disabledRecipeName);
```

Comment:
If we are dropping the `isDisabled` functionality this test is not as necessary anymore i dont think, as the first test should now be covering everthing.